atlasdb-config/src/test/java/com/palantir/atlasdb/factory/TransactionManagersTest.java

```diff
@@ -110,9 +110,9 @@
     private static final long EMBEDDED_BOUND = 3;
 
     private static final String TIMELOCK_SERVICE_FRESH_TIMESTAMP_METRIC =
-            MetricRegistry.name(TimelockService.class, "getFreshTimestamp");
+            MetricRegistry.name(TimelockRpcClient.class, "getFreshTimestamp");
```

Comment:
we will be breaking the metrics dashboard for client side - shall we put extra effort to keep metric name same by differing the class name and metric name? or would it cause more problems?

Reply:
nah; we can roll this out real quick and then it'll be the case for the whole fleet

Reply:
but i guess, given that this is just a string... why change it?